hawkbit-device-simulator/src/main/java/org/eclipse/hawkbit/simulator/http/AuthProperties.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) 2019 Bosch Software Innovations GmbH and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.hawkbit.simulator.http;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * Basic authentication properties.
+ *
+ */
+@ConfigurationProperties(prefix = "hawkbit.device.simulator.auth")
```

Comment:
try this one `prefix = OAuth2ClientResourceProperties.CONFIGURATION_PREFIX`

Reply:
Done:
renamed to BasicAuthProperties.java;
set prefix to BasicAuthProperties.CONFIGURATION_PREFIX